src/Topshelf/Configuration/HostConfigurators/HostConfigurator.cs

```diff
@@ -68,6 +68,11 @@ public interface HostConfigurator
         /// </summary>
         void EnableSessionChanged();
 
+        /// <summary>
+        /// Enabled support for power events (signaled by the host OS)
+        /// </summary>
+        void EnablePowerChanged();

```

Comment:
I believe that `EnablePowerEvents()` would be more descriptive.

Reply:
Done